build.sbt

```diff
@@ -20,6 +20,11 @@ lazy val databaseMigration = (project in file("database/migration"))
   .dependsOn(core)
   .withTestSettings
 
+lazy val dockerHashing = (project in file("dockerHashing"))
+  .settings(dockerHashingSettings: _*)
+  .dependsOn(core)
```

Comment:
I did manage to fight through most of the entanglement (see fb299b638a49418b9a4871098fd25c42ad2d2a52) with the exception of an exception: 

```
import cromwell.core.actor.StreamActorHelper.ActorRestartException
```

It looks like a common exception type would need to be used instead of this custom exception.  Also I can't guarantee there won't be more problems downstream of this.  😄